tests/resources/test_design_space.py

```diff
@@ -116,13 +117,76 @@ def test_create_default(valid_product_design_space_data,
 
     session = FakeSession()
     session.set_response(data_with_instance)
+    
+    predictor_id = uuid.uuid4()
     collection = DesignSpaceCollection(
         project_id=uuid.uuid4(),
         session=session
     )
-    default_design_space = collection.create_default(predictor_id=uuid.uuid4())
+
+    expected_call = FakeCall(
+        method='POST',
+        path=f"projects/{collection.project_id}/design-spaces/default",
+        json={
+            "predictor_id": predictor_id,
+            "include_ingredient_fraction_constraints": False,
+            "include_label_fraction_constraints": False,
+            "include_label_count_constraints": False
+        },
+        version="v2"
+    )
+    
+    default_design_space = collection.create_default(predictor_id=predictor_id)
+
+    assert session.num_calls == 1
+    assert session.last_call == expected_call
+    
     assert default_design_space.dump() == valid_product_design_space.dump()
 
+
+@pytest.mark.parametrize("config_flags", itertools.product((True, False), repeat=3))
+def test_create_default_with_config(valid_product_design_space_data, valid_product_design_space, config_flags):
+    ingredient_fractions, label_fractions, label_count = config_flags
```

Comment:
Wouldn't make more sense to:
```suggestion
@pytest.mark.parametrize("ingredient_fractions", (True, False))
@pytest.mark.parametrize("label_fractions", (True, False))
@pytest.mark.parametrize("label_count", (True, False))
def test_create_default_with_config(valid_product_design_space_data, valid_product_design_space, ingredient_fractions, label_fractions, label_count):
```

Reply:
*shrugs* Sure. They're the exact same, so I don't really care. As I'm making the tweak to the `*` formatting above, I'll make this change, too.